contracts/Interfaces/IPositionManager.sol

```diff
@@ -233,11 +252,61 @@ interface IPositionManager is IFeeCollector {
     /// @param collateralToken The token used as collateral.
     /// @return raftCollateralToken The Raft indexable collateral token.
     /// @return raftDebtToken The Raft indexable debt token.
+    /// @return priceFeed The contract that provides a price for the collateral token.
+    /// @return splitLiquidation The contract that calculates collateral split in case of liquidation.
     /// @return isEnabled Whether the collateral token can be used as collateral or not.
-    function raftCollateralTokens(IERC20 collateralToken)
+    /// @return lastFeeOperationTime Timestamp of the last operation for the collateral token.
+    /// @return borrowingSpread The current borrowing spread.
+    /// @return baseRate The current base rate.
+    /// @return redemptionSpread The current redemption spread.
+    /// @return redemptionRebate Percentage of the redemption fee returned to redeemed positions.
+    function collateralInfo(IERC20 collateralToken)
         external
         view
-        returns (IERC20Indexable raftCollateralToken, IERC20Indexable raftDebtToken, bool isEnabled);
+        returns (
+            IERC20Indexable raftCollateralToken,
+            IERC20Indexable raftDebtToken,
+            IPriceFeed priceFeed,
```

Comment:
These are immutable, so one can cache them locally in the external contract.
The others have their own getters in order to not read the whole struct in case external contracts need this information.